src/DetailsView/components/left-nav/common-left-nav-link.scss

```diff
@@ -11,7 +11,7 @@
 .left-nav-link-container {
     padding-left: 5%;
     padding-right: 8%;
-    width: $detailsViewLeftNavWidth;
+    width: 200px;
```

Comment:
This is maybe the most notable change in terms of "things that are probably a bug that the refactoring uncovered"; this was the only point where the pre-reflow `detailsViewLeftNavWidth` was still being used, and it's *not* consistent with the actual left nav width today. This PR is just maintaining the current behavior by inlining the old value in this one spot where it was still used, but this line will probably change to something less questionable in a followup PR to fix #4361